packages/kafka/kafka.0.4/url

```diff
@@ -0,0 +1,2 @@
+http: "https://github.com/didier-wenzek/ocaml-kafka/archive/0.3.3.tar.gz"
```

Comment:
Is this the correct archive? Why does it mismatch the version of the package submitted?

Reply:
I have been too lazy here => I will submit a fix.